tests/apm/mod.rs

```diff
@@ -2,10 +2,35 @@ use std::fs::{self, File};
 use std::io::{BufRead, BufReader};
 
 use bson::Bson;
-use mongodb::{Client, ClientOptions, CommandResult, ThreadedClient};
 use mongodb::db::ThreadedDatabase;
+use mongodb::{Client, ClientOptions, CommandResult, ThreadedClient};
 use rand;
 
+macro_rules! assert_eq_ignoring_ip {
```

Comment:
`ignoring_ip` isn't quite an accurate name.  It's agnostic to ipv4 or ipv6, but it still expects a localhost IP address.

Maybe `assert_eq_ipv4_or_ipv6`.